src/platforms/pumpfun/event_parser.py

```
@@ -0,0 +1,485 @@
+                        
+                        for ix in transaction.message.instructions:
+                            program_id = transaction.message.account_keys[ix.program_id_index]
+                            
+                            # Check if instruction is from pump.fun program
+                            if str(program_id) != str(self.get_program_id()):
+                                continue
+
+                            ix_data = bytes(ix.data)
+                            
+                            # Check for create discriminator
+                            if len(ix_data) >= 8:
```

Ruff (0.12.2)

Magic value used in comparison, consider replacing `8` with a constant variable

(PLR2004)